R/call.R

```diff
@@ -919,6 +919,13 @@ call_parse_type <- function(call) {
   .Call(rlang_which_operator, call)
 }
 call_has_precedence <- function(call, parent_call, side = NULL) {
+  side <- switch(
+    side %||% "none",
+    lhs = -1L,
+    none = 0L,
+    rhs = 1L,
```

Comment:
In `vec.h` isn't the `value == NA_LOGICAL` redundant if you have the `r_is_bool()` check?

```c
static inline
bool r_is_true(sexp* x) {
  if (!r_is_bool(x)) {
    return false;
  } else {
    int value = r_lgl_get(x, 0);
    return value == NA_LOGICAL ? 0 : value;
  }
}
```

Reply:
oops yes